backend/src/test/java/de/bund/digitalservice/ris/caselaw/integration/tests/FieldOfLawIntegrationTest.java

```diff
@@ -186,7 +186,11 @@ void testGetFieldsOfLawByNormsQuery_withoutAbbreviation(String query) {
   @ParameterizedTest
   @ValueSource(
       strings = {
+        "abc § 123", // norm followed by paragraph
+        "abc §123", // norm followed by paragraph without whitespace
+        "abc § 12", // norm followed by incomplete paragraph
         "§ 123 abc", // paragraph followed by norm
+        "§ 12 abc", // incomplete paragraph followed by norm
```

Comment:
praise: Nice to cover many edge cases. 👏  In general, these might hint that a unit test might be useful for the preprocessing, but an integration test on the database also works. (It does not start a new app for each parametrized test right?)